src/app/pages/OnDemandRadioPage/index.jsx

```diff
@@ -1,14 +1,53 @@
 import React, { useContext } from 'react';
-import { string, shape } from 'prop-types';
+import { string, number, shape } from 'prop-types';
 import styled from 'styled-components';
 import MetadataContainer from '../../containers/Metadata';
 import Grid, { GelPageGrid } from '#app/components/Grid';
 import { ServiceContext } from '../../contexts/ServiceContext';
-
 import HeadingBlock from '#containers/RadioPageBlocks/Blocks/Heading';
 import ParagraphBlock from '#containers/RadioPageBlocks/Blocks/Paragraph';
+import AudioPlayerBlock from '#containers/RadioPageBlocks/Blocks/AudioPlayer';
 
 const SKIP_LINK_ANCHOR_ID = 'content';
+const EPISODE_IS_AVAILABLE = 'is-available';
+const EPISODE_IS_EXPIRED = 'is-expired';
+const EPISODE_IS_NOT_YET_AVAILABLE = 'is-not-yet-available';
+
+const getTimeTicks = () => new Date().getTime() * 10000;
```

Comment:
Is 10,000 correct here?  If this is to convert to ms, I'd expect it to be 1000 🤔

Reply:
hmmmm I'll look into this. I'll maybe use momentjs to be sure I'm working with the correct time

Reply:
Fixed this @ryanmccombe